pkg/assets/stage.go

```diff
@@ -73,7 +71,19 @@ func Stage(dataDir string, name string, filemode os.FileMode) error {
 		return fmt.Errorf("failed to create dir %s: %w", filepath.Dir(p), err)
 	}
 
-	if ExecutableIsOlder(p) {
+	selfexe, err := os.Executable()
+	if err != nil {
+		logrus.Warn(err)
+		return err
```

Comment:
I would prefer to have here (and in other return err branches) fmt.Errorf, because it is often hard to find the exact codeline when we have some generic error message that's coming from the stdlibrary.

Reply:
Thanks. I cleaned up all of the returns here and added some consistency to the names of things logged (now all full-paths vs name-only)